frontend/src/scenes/session-recordings/player/list/PlayerList.tsx

```diff
@@ -0,0 +1,155 @@
+                            onClick={() => {
+                                scrollTo()
+                            }}
+                        >
+                            {isDirectionUp ? <ArrowUpOutlined /> : <ArrowDownOutlined />}
+                            Jump to current time
+                        </div>
+                        <div
+                            className="flex justify-center items-center right"
+                            onClick={() => {
+                                disablePositionFinder()
+                            }}
+                        >
+                            <CloseOutlined />
+                        </div>
+                    </div>
+                    <AutoSizer>
+                        {({ height, width }: { height: number; width: number }) => {
+                            return (
+                                <List
+                                    ref={listRef}
+                                    className="event-list-virtual"
+                                    height={height}
+                                    width={width}
+                                    onRowsRendered={setRenderedRows}
+                                    noRowsRenderer={() => (
+                                        <div className="event-list-empty-container">
+                                            <Empty
+                                                image={Empty.PRESENTED_IMAGE_SIMPLE}
+                                                description="No events fired in this recording."
+                                            />
+                                        </div>
+                                    )}
+                                    overscanRowCount={OVERSCANNED_ROW_COUNT} // in case autoscrolling scrolls faster than we render.
+                                    overscanIndicesGetter={({
+                                        cellCount,
+                                        overscanCellsCount,
+                                        startIndex,
+                                        stopIndex,
+                                    }) => {
+                                        const under = Math.floor(overscanCellsCount / 2)
+                                        const over = overscanCellsCount - under
+                                        return {
+                                            overscanStartIndex: Math.max(0, startIndex - under),
+                                            overscanStopIndex: Math.min(cellCount - 1, stopIndex + over),
+                                        }
+                                    }}
+                                    rowCount={data.length}
+                                    rowRenderer={({ index, style, key }) => {
+                                        const record = data[index] as T
+                                        const rowKeyDetermined = key ?? index
```

Comment:
nit: on first read, I'm not sure how to interpret the `Determimed` word here. Maybe a comment or change of name could help?

Reply:
I'm copying this terminology from LemonTable and TableRow, so I _think_ this should be fine.